avro/src/schema.rs

```diff
@@ -2782,7 +2788,7 @@ mod tests {
             ]
         }"#;
 
-        let schema_c = Schema::parse_list(&[schema_str_a, schema_str_b, schema_str_c])?
+        let schema_c = Schema::parse_list([schema_str_a, schema_str_b, schema_str_c])?
```

Comment:
The PR title says `feat` and `Accept more inputs ...` but I just tried to revert this change and check whether the old input is still allowed but it fails with:
```
error: the borrowed expression implements the required traits
    --> avro/src/schema.rs:2791:43
     |
2791 |         let schema_c = Schema::parse_list(&[schema_str_a, schema_str_b, schema_str_c])?
     |                                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: change this to: `[schema_str_a, schema_str_b, schema_str_c]`
     |
```
So it is actually an API break.

This is not a big problem because Avro-rs is still 0.x but it would be nice if the old API is still supported.

Reply:
Actually this was `clippy` !
The `build` is just fine!